src/components/com_tjdashboard/site/views/dashboards/tmpl/default.php

```diff
@@ -114,8 +114,7 @@
 
 						$canChange  = $this->canChangeStatus;
 						?>
-						<tr class="row
-							<?php echo $i % 2; ?>" sortable-group-id="
+						<tr class="row<?php echo $i % 2; ?>" sortable-group-id="
```

Comment:
Don't put only the attribute value on a new line. You can put the entire attribute and it's value on the next line

Reply:
done